testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -167,6 +168,45 @@ public void testSelectOnViewFromDifferentSchema()
         assertViewQuery("SELECT * FROM test_schema.hive_test_view_1", queryAssert -> queryAssert.hasRowsCount(25));
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testViewReferencingTableInDifferentSchema()
+    {
+        String schemaX = "test_schema_x" + randomTableSuffix();
+        String schemaY = "test_schema_y" + randomTableSuffix();
+        String tableName = "test_table_y" + randomTableSuffix();
+        String viewName = "view_schema_x" + randomTableSuffix();
+
+        onHive().executeQuery(format("DROP SCHEMA IF EXISTS %s CASCADE", schemaX));
```

Comment:
I don't think you need to do these `DROP ... IF EXISTS` when using `randomTableSuffix`, there shouldn't be collisions.